lib/subjects/postgresql_selection.rb

```diff
@@ -17,6 +17,15 @@ def select(options={})
       results.take(limit)
     end
 
+    def any_workflow_data(options={})
+      @opts = options
```

Comment:
Why is opts not just a constructor param? Query methods that change state (apart from caching) are icky, and it seems like all the code ever does is `PsqlSelection.new(workflow, user).select(opts)`, so why not just do `PsqlSelection.new(workflow, user, opts).select`?